mlflow/catboost.py

```diff
@@ -106,7 +106,7 @@ def save_model(
                         }
 
     :param mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
-    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+    :param signature: :py:class:`ModelSignature <mlflow.models.ModelSignature>`
```

Comment:
These are MLflow Models APIs. Can we move them to the other PR?

Reply:
Got it!